### ISSUE TITLE
jceminer needs config files like srbminer

### ISSUE THREAD

**Issue by jch9678** [RainbowMiner/RainbowMiner#313]
Would it be possible for you to adapt the procedure for benchmarking and running srbminer to jceminer.  jceminer needs config files to manually optimize gpus.  However, unlike srbminer you still need to pass the pool parameters in the bat file.  At least for me this is why benchmarking fails for jceminer.

**Comment by RainbowMiner** [comment]
Yes, of course this would be possible. I will figure this out.

**Comment by jch9678** [comment]
Sorry I'm confused....did you mean that you already implemented this for Srbminer and you will port it over to jceminer
or
did you mean that it is already implemented in jceminer. If this is the case it appears to be broken because there aren't config_[algorithm]-[devicename].txt files.

**Comment by RainbowMiner** [comment]
Oh, now I know: you refer to the JceminerAmd.
There is a config-API implemented for the JceminerCpu, only. The JceminerAmd runs with commandline, only, at this time.

**Comment by jch9678** [comment]
Sorry, I should have wrote jcemineramd.  Yes right now in RainbowMiner jcemineramd runs only with command line parameters but we need config files like srbminer has to optimize it.  Thank you for working on this.

**Comment by RainbowMiner** [comment]
I created a beta version to try out, if you like. It works like this: JceminerAmd will create config files with the name `config_JceminerAmd-[GPU#xx]-[Algorithm]-[Devicemodel].example.txt`, e.g. `config_JceminerAmd-GPU#00-GPU#01-CryptonightV8-RX550.example.txt`

- As long, as you do nothing, the parameter `--auto` will be added.
- If you copy one or more of the config files from `.example.txt` to `.txt`, Rainbowminer will detect this as manual config file and add it to Jceminer, using the `-c [configfilename].txt` parameter.

**Comment by jch9678** [comment]
Ok, so it's best to copy this file 'config_JceminerAmd-GPU#00-GPU#01-CryptonightV8-RX550.example.txt' and rename it to 'config_JceminerAmd-GPU#00-GPU#01-CryptonightV8-RX550.txt and make optimizations.  At this point RainbowMiner will see the new file, remove --auto, and use -c  Perfect  Thank You

**Comment by RainbowMiner** [comment]
Exactly 🌞

**Comment by RainbowMiner** [comment]
Here is the zip with the beta:
[Jcetest.zip](https://github.com/RainbowMiner/RainbowMiner/files/2578279/Jcetest.zip)

Stop Rainbowminer, extract the contents into the Rainbowminer folder. One file in APIs and one one file in Miners will be overwritten. Restart RainbowMiner.

If any errors pop up, please let me know asap.

**Comment by jch9678** [comment]
Ok I will erase the stats file and config files in the bin directory so there are 'clean' benchmarks.

**Comment by jch9678** [comment]
1st stage worked great, I got 
config_JceminerAmd-GPU#00-CryptoNightV8-R9290X.example.txt
config_JceminerAmd-GPU#00-CryptoNightV7-R9290X.example.txt
config_JceminerAmd-GPU#00-CryptoNightHeavy-R9290X.example.txt

and 2nd stage success as well.  I changed a multi hash value for v8 in the new file without 'example', deleted the v8 stats file, and it re-benchmarked v8 with the updated optimization.

So cool.
